File: java/src/main/java/org/example/deck/DeckRun.java
package org.example.deck;

public class DeckRun {
    public static void run() {
        var deck = new Deck();

        deck.pushFront("test2");
        deck.pushFront("house1");
        deck.pushBack("home3");
        deck.pushBack("work4");
        deck.show();

        System.out.println("\n\n");

        System.out.println(deck.front());
        System.out.println(deck.back());

        System.out.println("\n\n");

        deck.popBack();
        System.out.println(deck.front());
        System.out.println(deck.back());

        System.out.println("\n\n");

        deck.popFront();
        System.out.println(deck.front());
        System.out.println(deck.back());

        System.out.println("\n\n");

        deck.show();
    }
}

File: php/src/Deck/DeckRun.php
<?php

namespace Claudiosilvajunior\Php\Deck;

class DeckRun
{
    public static function run(): void
    {
        $deck = new Deck();

        $deck->pushFront("mwwmo");
        $deck->pushFront(02);
        $deck->pushBack(12);
        $deck->pushBack("dqdw");
        $deck->show();

        print_r($deck->front());
        print_r("\n");
        print_r($deck->back());

        $deck->popBack();
        print_r("\n");
        print_r($deck->front());
        print_r("\n");
        print_r($deck->back());

        $deck->popFront();
        print_r("\n ");
        print_r($deck->front());
        print_r("\n ");
        print_r($deck->back());
        print_r("\n");
        $deck->show();
    }
}
